p2panda-rs/src/cddl/generator.rs

```diff
@@ -123,17 +132,26 @@ mod tests {
         person.insert("height".to_string(), FieldType::Float);
         person.insert("is_cool".to_string(), FieldType::Bool);
         person.insert("favorite_food".to_string(), FieldType::Relation);
+        person.insert("top_ten_foods".to_string(), FieldType::RelationList);
+        person.insert("one_specific_meal".to_string(), FieldType::PinnedRelation);
+        person.insert(
+            "top_ten_specific_meals".to_string(),
```

Comment:
Don'y you change my mother's oat meal recipe!!

Reply:
Pin it!

Reply:
:rofl: